### PR TITLE
RA-1807:Modifiers should be declared in the correct order

### PR DESCRIPTION
**Pull request by Wandji69**
Link to the JIRA ticket: https://issues.openmrs.org/browse/RA-1807

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java [hunk 1]
```diff
@@ -54,10 +54,10 @@
  */
 public class RegisterPatientFormBuilder {
 
+	public static final String DEMOGRAPHICS_SECTION_ID = "demographics";
+
 	protected final static Log log = LogFactory.getLog(RegisterPatientFormBuilder.class);
 
-	public static final String DEMOGRAPHICS_SECTION_ID = "demographics";
-
```
Comment (sacull) [inline omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java:60]: image
Why did you move the fragment marked 3 to the position marked 1 and left what you should change, which is the position marked 2?

**Review by sacull — CHANGES_REQUESTED** [review]

File: omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java (hunk 1 above)
Comment (Wandji69) [reply omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java:60]: I have made the requested changes and for the change of position, I just thought it will be more okay declaring the public first before the protected.  

> image
 Why did you move the fragment marked 3 to the position marked 1 and left what you should change, which is the position marked 2?

Comment (sacull) [reply omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java:60]: We try not to modify what is not the subject of the ticket. If something needs to be changed in the code, we try to create a separate ticket for it. ;)

**Comment by sacull** [comment]
If you change something that is not in the subject of the ticket, you introduce some confusion to the person who checks the corrections, because what has been corrected does not match what the ticket describes.

Ideally, GitHub should only mark the lines that are the subject of the ticket.

**Comment by Wandji69** [comment]
> If you change something that is not in the subject of the ticket, you introduce some confusion to the person who checks the corrections, because what has been corrected does not match what the ticket describes.
> 
> Ideally, GitHub should only mark the lines that are the subject of the ticket.

Okay understood, will fix that right away.

**Comment by Wandji69** [comment]
Updates made as requested.

**Review by sacull — APPROVED** [review]
It's very good for me now. ;)
